docs/operator-manual/applicationset/Generators-SCM-Provider.md

```diff
@@ -255,6 +255,84 @@ This SCM provider does not yet support label filtering
 
 Available clone protocols are `ssh` and `https`.
 
+## AWS CodeCommit (Alpha)
```

Comment:
We should probably mention that the `sha` output param isn't supported.

Reply:
addressed